examples/multiple-accounts/packages/contracts/package.json

```
@@ -28,7 +28,7 @@
     "prettier": "3.2.5",
     "prettier-plugin-solidity": "1.3.1",
     "solhint": "^3.3.7",
-    "solhint-config-mud": "2.0.0-next.15",
-    "solhint-plugin-mud": "2.0.0-next.15"
+    "solhint-config-mud": "2.0.0-next.16",
+    "solhint-plugin-mud": "2.0.0-next.16"
```

nothing changed but good to be on the latest version
```suggestion
    "solhint-config-mud": "2.0.4",
    "solhint-plugin-mud": "2.0.4"
```